2252 change validation sidebar header text for the examples

jonfroehlich opened the pull request:
Resolves https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2252. Updates both the look and text of the "Example" headers on the right sidebar.

##### Before/After screenshots
**Before:**
![image](https://user-images.githubusercontent.com/1621749/94082797-9aba5780-fdb6-11ea-80b7-8d8b331e3888.png)

![image](https://user-images.githubusercontent.com/1621749/94082821-a86fdd00-fdb6-11ea-818d-f89bc5efc07d.png)

**After:**
![image](https://user-images.githubusercontent.com/1621749/94082769-88d8b480-fdb6-11ea-9cac-6e0ffb8a499a.png)

![image](https://user-images.githubusercontent.com/1621749/94082864-b6bdf900-fdb6-11ea-989d-41e1142a6405.png)


##### Things to check before submitting the PR <!-- if something doesn't apply, just check the box or remove the line -->
- [x] If there is a visual design component, I've posted before/after screenshots on the relevant issue thread and have been given a thumbs up on my design.
- [x] I've asked for and added translations for any user facing text that was added or modified.
- [x] I've made sure my code and comments follow the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide).
- [x] I've tested on multiple browsers.

##### Testing instructions
1. Just load any validation page

misaugstad commented:
@jonfroehlich this is what the sidebar looks like when I check out your branch
![Screenshot from 2020-09-25 19-49-20](https://user-images.githubusercontent.com/6518824/94328541-46ef7000-ff68-11ea-96a5-21e6e8e1e585.png)

jonfroehlich commented:
I just checked and that's not what it looks like for me. Did you try clearing your cache?

![image](https://user-images.githubusercontent.com/1621749/94340202-b68b4c80-ffb4-11ea-864c-8148b00eadbb.png)

![image](https://user-images.githubusercontent.com/1621749/94340220-e20e3700-ffb4-11ea-9c79-a75d47546ceb.png)

jonfroehlich commented:
Hey @misaugstad, I'd like to get this fixed and merged but I'm not seeing the problem you described. Can you test again and then provide me with details on how to replicate?

misaugstad commented:
Not sure why they didn't show up for you. I didn't test again just now, but I peeked at the code, and it looks like it has to be from the code you (I assume accidentally) added to validation.scala.html

jonfroehlich commented:
I just re-looked at my code, and I don't see anything wrong in there tbh. And it's not rendering incorrectly on my local dev machine.

misaugstad commented on app/views/validation.scala.html:
```diff
@@ -185,6 +185,7 @@ <h2 id="total-mission-reward" style="display:none"></h2>
                 </div>
                 <div class="status-box">
                     <h1 class="status-holder-header-1" id="label-type-example"></h1>
+                    <strong>Correct</strong> Examples
```
Did you mean to add these?

jonfroehlich commented:
Thanks @misaugstad for the phone call to help explain my mistake. I moved stuff to messages. And fixed some minor aesthetic things. Ready for another review.

![image](https://user-images.githubusercontent.com/1621749/94746813-699bd300-0332-11eb-9f3e-7e15102e5018.png)

![image](https://user-images.githubusercontent.com/1621749/94810005-50356e00-03a8-11eb-9804-d5b6f99f8bc5.png)

misaugstad commented:
@jonfroehlich I don't know why you were not seeing the bug, but it was still there for me. The issue was that you added the updated text but you didn't remove the old text. You can see in my last two commits where I remove the old text.

jonfroehlich commented:
My guess is I either forgot to commit my most recent changes or I was accidentally editing the file in /target, which has happened too often recently (and I don't notice unfortunately). But I had definitely solved the problem on my local dev machine.

(Note I've more recently realized that if a filename shows in orange in its editor tab then IntelliJ IDEA is indicating that it's coming from the /target area).
